app/src/main/java/org/fossasia/openevent/general/order/OrderDetailsViewHolder.kt

```diff
@@ -22,5 +26,12 @@ class OrderDetailsViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView)
         itemView.orderIdentifier.text = orderIdentifier
         itemView.eventSummary.text = event?.description
         itemView.organizer.text = event?.organizerName
+
+        try {
+            val bitMatrix = MultiFormatWriter().encode(orderIdentifier, BarcodeFormat.QR_CODE, 200, 200)
```

Comment:
Create a model and encapsulate the logic in it

There should be no such code in ViewModel which cannot be unit tested

Reply:
@srv-twry Can you confirm the barcode content

Reply:
![image](https://user-images.githubusercontent.com/21277837/43506029-6d046faa-9586-11e8-8227-a54a5362aae8.png)

The online QR code reader detects "simarsingh24@gmail.com".

Reply:
@srv-twry we are generating it on order identifier the attached screen shot is different

Reply:
@srv-twry I meant the QR code content is what Orga App is expecting or not...